src/types/types.ts

```diff
@@ -1,3 +1,5 @@
+// in review
```

Comment:
Suggest putting each type/enum into its own file. It makes a lot of files, but it makes the code easier to navigate.